### PR TITLE
Update MapETA translations in zh-CHT

### PR DESCRIPTION
**Pull request by eFiniLan**

### CODE REVIEW

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 1]
```diff
@@ -355,35 +355,35 @@
...
         <location filename="../qt/maps/map.cc" line="421"/>
         <source> km</source>
-        <translation> km</translation>
+        <translation> 公里</translation>
     </message>
     <message>
         <location filename="../qt/maps/map.cc" line="424"/>
         <source> m</source>
-        <translation> m</translation>
+        <translation> 公尺</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_zh-CHT.ts:376]: Are the Chinese translations more recognizable than the original English text for these abreviations? Weixing said km is more recognizable for simplified

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 2]
```diff
@@ -1020,23 +1020,23 @@ location set</source>
     <message>
         <location filename="../qt/offroad/settings.cc" line="259"/>
         <source>Switch Branch</source>
-        <translation type="unfinished"></translation>
+        <translation>切換分支</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_zh-CHT.ts:1023]: Thanks for adding these!

File: selfdrive/ui/translations/main_zh-CHT.ts [hunk 3]
```diff
@@ -340,7 +340,7 @@
     <message>
         <location filename="../qt/maps/map.cc" line="622"/>
         <source>eta</source>
-        <translation>埃塔</translation>
+        <translation>預計到達</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_zh-CHT.ts:343]: This might overflow in the UI

Comment (eFiniLan) [reply selfdrive/ui/translations/main_zh-CHT.ts:343]: yeah that's a problem, I've already shortened it from "預計到達時間" to "預計到達"

File: selfdrive/ui/translations/main_zh-CHT.ts (hunk 1 above)
Comment (eFiniLan) [reply selfdrive/ui/translations/main_zh-CHT.ts:376]: true, I'll revert them :)

**Pull request merged**